packages/notification-service/app.alfajoresstaging.yaml

```diff
@@ -0,0 +1,6 @@
+runtime: nodejs
```

Comment:
Since we're using the existing staging Firebase DB for this, wondering if it'd be simpler to also just use teh existing app.staging.yaml here?